scripts/RollTableMagicSurge.js

```diff
@@ -18,7 +18,7 @@ export async function RollTableMagicSurge() {
   );
 
   // TODO: Remove in final v0.8 release
-  if (parseInt(game.data.version.replaceAll(".", "")) > 80) {
+  if (parseInt(game.data.version.replace(/./g, '')) > 80) {
```

Comment:
Thanks so much for raising this PR, really appreciate the feedback and taking the time to put a fix in rather than a bug ticket. Will get a contributors.md added soon to add you.

Minor linting thing, can you change `''` to `""` to get it pass the checks please then I will approve, merge and release.